sstables_loader.cc

```diff
@@ -798,7 +853,7 @@ future<> sstables_loader::download_task_impl::run() {
 
     if (ex) {
         co_await _loader.invoke_on_all([&sstables_on_shards, &erms] (sstables_loader&) {
-             // clear on correct shard
+            // clear on correct shard
```

Comment:
Might have happened in https://github.com/scylladb/scylladb/pull/26469/commits/575bcf1acf1ee8d9c3abc8a556198e07e284643a